Exiled.API/Features/Player.cs

```diff
@@ -1955,5 +1955,11 @@ public void ChangeEffectIntensity(EffectType type, byte intensity, float duratio
 
         /// <inheritdoc/>
         public override string ToString() => $"{Id} {Nickname} {UserId} {Role} {Team}";
+
+        /// <summary>
+        /// Sets noclip status to a specific player.
+        /// </summary>
+        /// <param name="status">The value to be set.</param>
```

Comment:
The value to be set -> Whether or not noclip should be activated.

Reply:
Could be just Whether noclip should be activated.

Reply:
^^